Tests/Fixtures/TestProject/project.yml

```
@@ -113,12 +113,16 @@ targets:
       PRODUCT_BUNDLE_IDENTIFIER: com.project.app
     dependencies:
       - target: Framework_iOS
+        platform: mac+iOS
       - target: StaticLibrary_ObjC_iOS
       - carthage: Result
+        platform: mac
```

if we change to macOS
```suggestion
        platform: macOS
```

---

Updated in dfc471d8511f37738bfb9a84915d822bcc299f1e.